src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_BALL_NUMBER = 100;
+
+    public Ball getRandomBall() {
+        Random random = new Random();
```

Comment:
Think about which variables should be local in the method and which should be class-level. new Random() object will be created on each getRandomBall() call. How can you avoid this?